_posts/2020-09-30-omero-5-6-3.md

```diff
@@ -7,7 +7,7 @@ Today we are releasing OMERO.server 5.6.3 which
 uses OMERO.blitz 5.5.8 to expand the
 OMERO API's `Duplicate` functionality to allow the duplication of
 images, attachments, and other model objects that are backed by files on
-disk. This long-awaited ability to create a true copy of microscopy data can be
+disk. Duplicated files may be linked with originals to save space on underlying storage. This long-awaited ability to create a true copy of microscopy data can be
```

Comment:
I am not sure this is what @joshmoore meant with ``mention... links versus copy`` I would go for something like

```
Previously, Images in OMERO could not be truly copied, the only possiblity was to create new links to other objects such as Datasets. Since version .... the full duplication workflow is offered, which gives the user the possiblity jto create Image duplicates independent of the original image.
```

But actually, I would add it to what you added here, as this is valid and useful as well imho

Reply:
The "link to the doc" @joshmoore might have meant could be https://omero-guides.readthedocs.io/en/latest/introduction/docs/data-management.html?highlight=warning#manage-images-in-datasets-projects - this is an existing feature which does not change in this release - but as @mtbc remarked, the doc is a bit out-of-date as we speak, as https://github.com/ome/omero-guide-introduction/pull/15 should improve (but not change) that paragraph (the confusing bit might be that the old doc link suggests "no other copy than copy link is possible in OMERO" whcih is precisely changed in https://github.com/ome/omero-guide-introduction/pull/15